### DISCUSSION TITLE
Schedules not working

### DISCUSSION THREAD

**Discussion by RayCodeCSC** [vilhelmprytz/pterodactyl-installer#309] [Q&A]
Hello, schedules not working. Can somebody help me please?

[pteroq.service](https://i.imgur.com/xs4GuuP.png)

I tried this: https://github.com/pterodactyl/panel/issues/3876#issuecomment-1013653476
But still not working.

**Comment by Linux123123** [comment]
```bash
/usr/bin/php /var/www/pterodactyl/artisan queue:work --queue=high,standard,low --sleep=3 --tries=3
```

What is the output?

**Reply by RayCodeCSC** [reply]
What you mean? How can i check the output?

**Comment by RayCodeCSC** [comment]
I think is fixed :D . Just reinstall the server.

**Comment by RayCodeCSC** [comment]
Okay so, not fixed :D . I think is it caused by changing permissions to 0777 of different server files.

I have installed CS 1.6 server, and if i add new plugins or something like this (copying files), then chmod them to 0777 <- this causing the problem i think. But i'm not sure.

**Comment by RayCodeCSC** [comment]
Okay, now fixed :D .

**Solution:**
I had installed PHP 8.0, and i was install SSH2 for PHP 7.0 :D .

So, if anybody has similar problem with pteroq as i'm, check your PHP extensions versions. Make sure, that you have installed all extensions for right PHP version,
